### PR TITLE
Dogfood 6-14-22

### PR DESCRIPTION
**Pull request by aarondr77**
# Description

- Adds `import from external drive` to upgrade to pro taskpane
- Fixes log anonymization with fillna. It adds `type` to the list of public params. TODO: `value` is a duplicated param name that is used to receive user emails, so we cannot remove it. 
- Fixes bug where the merge taskpane turns into the import taskpane that says you need to load more data when the next sheet is empty. 
- Removes all disabled checks for data and formula columns, since they don't exist anymore!
- Adds icons for the step history taskpane for changing dtypes, promote row to header, deleting columns, split text to columns, and tranpose. 
- Fixes spacing of toggle in cell editor on notebooks
- Widens the tour so that text is not cut off
- Fixes bug where the drive icon was not displayed on notebooks on Macs. 

# Testing

Test each of the changes + the new documentation. 

# Documentation

There is [new documentation ](https://app.gitbook.com/o/-MS4px_3crWEYYdbNPU8/s/-MP_U5ZCmiamDOXEOOTC/~/changes/jWM0mxZNcfJ3OaLdeI2w/how-to/transpose) for Transpose, Column Formulas, and Importing.

### CODE REVIEW

**Comment by aarondr77** [comment]
When we transpose a date, it doesn't remain formatted as a date.

**Comment by aarondr77** [comment]
The tour is cut off in the notebook

Screen Shot 2022-06-14 at 12 56 54 PM

File: mitosheet/src/components/endo/ColumnHeaderDropdown.tsx [hunk 1]
```diff
@@ -139,11 +138,10 @@ export default function ColumnHeaderDropdown(props: {
                         // As in google sheets, if the starting formula is non empty, we default to the 
                         // arrow keys scrolling in the editor
                         arrowKeysScrollInFormula: columnFormula !== undefined && columnFormula.length > 0,
-                        editorLocation: 'cell'
+                        editorLocation: 'cell',
+                        editingMode: 'set_column_formula'
                     })
```
Comment (naterush) [inline mitosheet/src/components/endo/ColumnHeaderDropdown.tsx:143]: Check this file for set cell value

File: mitosheet/src/components/endo/celleditor/CellEditor.tsx [hunk 2]
```diff
@@ -492,22 +491,35 @@ const CellEditor = (props: {
...
+                                        ...prevEditorState,
+                                        editingMode: prevEditingMode === 'set_column_formula' ? 'set_cell_value' : 'set_column_formula'
+                                    }
                                 })
                             }}
                             height='20px'
                         />
                     </Row>
                 }
+                {cellEditorError === undefined && props.editorState.rowIndex == -1 &&
```
Comment (naterush) [inline mitosheet/src/components/endo/celleditor/CellEditor.tsx:518]: Is the cell editor ever open for a rowIndex <= -1? I think no.

**Review by naterush — COMMENTED** [review]
Test and look into these things.

**Comment by naterush** [comment]
@aarondr77 can you add a test for that convert None -> False, and then LGTM. I made a few tiny changes above + to the docs, but everything looks great otherwise. 

Nice thorough review! Lots of good work here really polishing things :-)
